### PR TITLE
Use uploadable profile pictures!

### PR DESCRIPTION
**Pull request by jarednorman**

### CODE REVIEW

File: app/assets/stylesheets/players/_edit.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.player-edit-form {
+  max-width: $main-area-width;
+  margin: 0 auto;
+  padding: $main-area-outer-padding 1rem;
+
+  .input.string {
```
Comment (jhawthorn) [inline app/assets/stylesheets/players/_edit.scss]: Should there be something here?

File: app/assets/stylesheets/players/_edit.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  }
+}
+
+.player-edit-profile-picture-fields {
```
Comment (jhawthorn) [inline app/assets/stylesheets/players/_edit.scss]: Should there be something here?

File: app/uploaders/profile_picture_uploader.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
+class ProfilePictureUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
+
+  if ENV['AWS_ACCESS_KEY_ID'] && ENV['AWS_SECRET_ACCESS_KEY']
+    storage :fog
+  else
+    storage :file
```
Comment (jhawthorn) [inline app/uploaders/profile_picture_uploader.rb:9]: Do we need a `.gitignore` entry to cover this?

**Review by jhawthorn — DISMISSED** [review]

**Review by forkata — APPROVED** [review]
:ping_pong:

**Comment by jarednorman** [comment]
Need to add a change so that players can access their "old" avatars so they can upload them if they want.

**Pull request merged**